development/tools/jobs/tester/buildjob/jobs.go

```diff
@@ -0,0 +1,168 @@
+package buildjob
+
+import (
+	"fmt"
+	. "github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/test-infra/prow/config"
+	"path"
+	"strings"
+	"testing"
+)
+
+type Suite struct {
+	path              string
+	repository        string
+	image             string
+	releases          []*SupportedRelease
+	runIfChanged      string
+	runIfChangedCheck string
+	fileSuffix        string
+	noMaster          bool
```

Comment:
maybe add a comment for that, it's not so descriptive. AFAIK it refers to jobs which were removed from the master but are still available on supported releases?

same thing for `runIfChanged`, runIfChangedCheck`, `fileSuffix`

Reply:
Yes, it says if the test should expect jobs for the master branch.

I will try to rename fields rather then putting comments on them.